modules/garbage-collection.adoc

```diff
@@ -0,0 +1,86 @@
+= {productname} garbage collection
+[[garbage-collection]]
+
+== About {productname} garbage collection
+
+{productname} includes automatic and continuous image garbage collection, which ensures efficient use of resources for active objects by removing objects that occupy sizeable amounts of disk space, like dangling or untagged images, repositories, and blobs, including layers and manifests. Garbage collection performed by {productname} can reduce costs without requiring scheduled downtime or read-only mode, and increase an organization's security by removing unused objects in their filesystem.
+
+== {productname} garbage collection in practice
+
+
+
+
+
+== Types of garbage collection
+
+With {productname}, you can set the following features are available to enable, or disable, garbage collection on image tags, namespaces, and repositories.
+
+[options="header", width=100%, cols=".^2a,.^3a,.^9a,.^4a"]
+|===
+|Name |Description |Schema
+| **FEATURE_GARBAGE_COLLECTION** | Wether garbage collection is enabled for image tags. Defaults to `true`. |Boolean
+| **FEATURE_NAMESPACE_GARBAGE_COLLECTION** | Whether garbage collection is enabled for namespaces. Defaults to `true`. |Boolean
+| **FEATURE_REPOSITORY_GARBAGE_COLLECTION** | Whether garbage collection is enabled for repositories. Defaults to `true`. Boolean.
+|===
+
+== Disabling garbage collection
+
+The garbage collection features for image tags, namespaces, and repositories are stored in the `config.yaml`. These feature default to `true`.
+
+Disabling garbage collection is not recommended. If disabled, dangling or untagged images, repositories, namespaces, layers, and manifests are not removed. This can increase costs, downtime, and security concerns.
+
+In rare cases, you might want to disable garbage collection, for example, to manually control when garbage collection is performed. You can disable garbage collection by setting the `GARBAGE_COLLECTION` features to `false`.
```

Comment:
We have additional flags that allows configuring the various timeouts, frequency at which the workers run, etc... I don't think we currently document these, but in general users shouldn't have to fiddle with these.